src/openrct2/ride/Vehicle.h

```diff
@@ -274,6 +276,12 @@ struct rct_vehicle : rct_sprite_common
     void Invalidate();
     void SetState(VEHICLE_STATUS vehicleStatus, uint8_t subState = 0);
     bool IsGhost() const;
+    void UpdateSoundParams() const;
+
+private:
+    bool SoundCanPlay() const;
```

Comment:
All the others are like spoken word `GetSoundPriority`, `UpdateSoundParams`, `CreateSoundParam`. This one is a bit Yoda-ish with `SoundCanPlay`, I guess? Just a random comment

Reply:
It makes sense if you put it into an if statement, though: 
```C++
if (SoundCanPlay)
    doStuff();
```

Reply:
But then it would sound even better, like a question, if it was `CanPlaySound`, no?
```c++
if (CanPlaySound())
    doStuff();
```

Reply:
I was originally going to prefix all of these sound functions so that it was SoundXXXX. I still don't really like the name anyway and will think of something better.

Reply:
@tupaschoal That sounds less like a question, IMO. (Or at least: less like an English question. It might be different for Portuguese.)

Reply:
Hmm, It's no big deal anyway, I think the message gets through on both, and I'm not 100% happy with either :P